test/cli/gateway_test.go

```diff
@@ -28,7 +29,7 @@ func TestGateway(t *testing.T) {
 	client := node.GatewayClient()
 	client.TemplateData = map[string]string{
 		"CID":    cid,
-		"PeerID": node.PeerID().String(),
+		"PeerID": peer.ToCid(node.PeerID()).String(),
```

Comment:
make this and all other places base36, so users are not confused by different CID for IPNS Record.

(base36 is the only thing that fits in a DNS label, and is already used by `ipfs name` commands